app/templates/suppliers/registered_address.html

```diff
@@ -0,0 +1,147 @@
+{% extends "_base_page.html" %}
+{% import "macros/forms.html" as forms %}
+{% import "toolkit/summary-table.html" as summary %}
+{% block head %}
+  <link rel="stylesheet" href="{{ asset_path }}stylesheets/location-autocomplete.min.css" />
+  {{ super() }}
+{% endblock %}
+{% block page_title %}Registered address – Digital Marketplace{% endblock %}
+
+{% block breadcrumb %}
+  {%
+    with items = [
+      {
+        "link": "/",
+        "label": "Digital Marketplace"
+      },
+      {
+        "link": url_for(".dashboard"),
+        "label": "Your account"
+      },
+      {
+        "link": url_for(".supplier_details"),
+        "label": "Supplier details"
+      },
+    ]
+  %}
+    {% include "toolkit/breadcrumb.html" %}
+  {% endwith %}
+{% endblock %}
+
+
+{% block main_content %}
+  {% if error %}
+  <div class="validation-masthead" aria-labelledby="validation-masthead-heading">
+    <h3 class="validation-masthead-heading" id="validation-masthead-heading">
+      {{ error }}
+    </h3>
+  </div>
+  {% endif %}
+
+  {% if registered_address_form.errors or registered_country_form.errors %}
+      <div class="validation-masthead" aria-labelledby="validation-masthead-heading">
+          <h3 class="validation-masthead-heading" id="validation-masthead-heading">
+              There was a problem with the details you gave for:
+          </h3>
+          {% if registered_address_form.errors %}
+          <ul>
+          {% for field_name, field_errors in registered_address_form.errors|dictsort %}
+            {% for error in field_errors %}
+              <li><a href="#{{ field_name }}" class="validation-masthead-link">{{ registered_address_form[field_name].label.text }}</a>
+            {% endfor %}
+          {% endfor %}
+          </ul>
+          {% endif %}
+
+          {% if registered_country_form.errors %}
+          <ul>
+          {% for field_name, field_errors in registered_country_form.errors|dictsort %}
+            {% for error in field_errors %}
+            <li><a href="javascript:document.getElementById('location-autocomplete').focus()" class="validation-masthead-link">{{ registered_country_form[field_name].label.text }}</a></li>
+            {% endfor %}
+          {% endfor %}
+          </ul>
+          {% endif %}
+      </div>
+  {% endif %}
+
+  <div class="grid-row">
+    <div class="column-one-whole">
+      {% with
+        heading = "What is your registered office address?",
+        smaller = true
+      %}
+        {% include 'toolkit/page-heading.html' %}
+      {% endwith %}
+    </div>
+  </div>
+
+  <form action="{{ url_for('.update_registered_address') }}" method="post" enctype="multipart/form-data">
```

Comment:
Do we need this enctype? I think we only need it if there is a file upload which I don't think there is for this form?

https://stackoverflow.com/questions/4526273/what-does-enctype-multipart-form-data-mean

Reply:
Yeah, I think you're right. That link is an interesting read too!